docs/api/paths/admin/db-_offline.yaml

```diff
@@ -10,7 +10,9 @@ parameters:
 post:
   summary: Take the database offline
   description: |-
-    This will take the database offline meaning actions can be taken without disrupting current operations ungracefully or having the restart the Sync Gateway instance.
+    If using persistent config, call [POST /{db}/_config](#operation/post_db-_config) with `{"offline": true}` to set the database to offline.
+
+    This will take the database offline on this node only. Actions can be taken without disrupting current operations ungracefully or having the restart the Sync Gateway instance.
```

Comment:
This reads as though the 2nd paragraph is the result of calling `POST /{db}/_config` since it kind of just runs on into it.

I think the `/db/_online` one is worded better since the order of paragraphs is reversed, and would advise the same here.